docs/tasks/administer-cluster/kubeadm-upgrade-1-7.md

```diff
@@ -0,0 +1,56 @@
+---
+assignees:
+- pipejakob
+title: Upgrading kubeadm clusters from 1.6 to 1.7
+redirect_from:
+- "/docs/admin/kubeadm-upgrade-1-7/"
+- "/docs/admin/kubeadm-upgrade-1-7.html"
+---
+
+* TOC
+{:toc}
+
+This guide is for upgrading kubeadm clusters from version 1.6.x to 1.7.x.
+Upgrades are not supported for clusters lower than 1.6, which is when kubeadm
+became Beta.
+
+**WARNING**: These instructions will **overwrite** all of the resources managed
+by kubeadm (static pod manifest files, service accounts and RBAC rules in the
+`kube-system` namespace, etc.), so any customizations you may have made to these
+resources after cluster setup will need to be reapplied after the upgrade. The
+upgrade will not disturb other static pod manifest files or objects outside the
+`kube-system` namespace.
+
+1. Back up `/etc/kubernetes`.
+
+2. Upgrade system packages.
+
+   Upgrade your OS packages for kubectl, kubeadm, kubelet, and kubernetes-cni.
+
+   a. On Debian, this can be accomplished with:
+
+       sudo apt-get update
+       sudo apt-get upgrade
```

Comment:
My debian is rusty, but on fedora/*EL you don't need the extra step so I'll comment below.